### PR TITLE
New package: goodvibes-0.3.7

### LINKED ISSUE #413: Package Request: goodvibes
### System

* xuname:  
  *Void 4.16.14_1 x86_64-musl GenuineIntel uptodate rrn*

### Website
https://gitlab.com/goodvibes/goodvibes

### Package on other linux distributions
https://aur.archlinux.org/packages/goodvibes/

### PR DESCRIPTION
**Pull request by Johnnynator**
resolves #413

### CODE REVIEW

**Comment by cr6git** [comment]
Also closes [#14612](https://github.com/voidlinux/void-packages/pull/14612)

**Comment by Johnnynator** [comment]
xD. I probably could have checked if there is already a PR on the old Repo. Any reason why you disabled the dbus server at compile time? It can be disabled in the settings at runtime.

**Comment by cr6git** [comment]
No, I don't remember why I disabled dbus on my template...

File: srcpkgs/goodvibes/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'goodvibes'
+pkgname=goodvibes
+version=0.3.7
+revision=1
+wrksrc="goodvibes-v${version}"
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel pkg-config glib-devel"
+makedepends="dconf-devel glib-devel gst-plugins-base1-devel gtk+3-devel
+ libkeybinder3-devel libsoup-devel gettext-devel"
```
Comment (cr6git) [inline srcpkgs/goodvibes/template]: Any reason why you put `gettext-devel` and `glib-devel` in both `hostmakedepends` / `makedepends` ?

Comment (Johnnynator) [reply srcpkgs/goodvibes/template]: They contain some binaries that are executed while compiling. But they are probably not needed for makedepends, at least gettext not. Will recheck it.

Comment (Johnnynator) [reply srcpkgs/goodvibes/template]: I removed them form makedepends since they are not needed there.

**Pull request merged**